client/packages/common/src/ui/components/navigation/Tabs/DetailTabs.tsx

```diff
@@ -17,6 +18,15 @@ export const DetailTabs: FC<DetailTabsProps> = ({ tabs }) => {
   const [currentTab, setCurrentTab] = useState<string>(tabs[0]?.value ?? '');
   const t = useTranslation('common');
 
+  const { isOpen } = useDetailPanelStore();
+
+  // Ugly hack to force the "Underline" indicator for the currently active tab
+  // to re-render when the "More" details panel is expanded. See issue #777 for
+  // more detail.
+  useEffect(() => {
+    window.dispatchEvent(new Event('resize'));
```

Comment:
you didn't need to debounce? For me the underline didn't quite end up in the right place without it

Reply:
I get this after opening and closing a couple of times:

![Screenshot 2022-12-19 at 5 25 55 PM](https://user-images.githubusercontent.com/9192912/208347508-6dd0bfda-4269-4333-8831-dc9619c661c2.png)

Reply:
Yeah, it looked fine with no debouncing for me (using Firefox).

I'll play round with it in different browsers I guess and find something that works for all.

Reply:
ah, right. I'm using chrome 🙄